contracts/proxies/GnosisSafeProxy.sol

```diff
@@ -17,28 +18,26 @@ contract GnosisSafeProxy {
 
     /// @dev Constructor function sets address of master copy contract.
     /// @param _masterCopy Master copy address.
-    constructor(address _masterCopy)
-        public
-    {
+    constructor(address _masterCopy) {
         require(_masterCopy != address(0), "Invalid master copy address provided");
         masterCopy = _masterCopy;
     }
 
     /// @dev Fallback function forwards all transactions and returns all received return data.
-    function ()
+    receive ()
         external
         payable
     {
         // solium-disable-next-line security/no-inline-assembly
         assembly {
-            let masterCopy := and(sload(0), 0xffffffffffffffffffffffffffffffffffffffff)
-            // 0xa619486e == keccak("masterCopy()"). The value is right padded to 32-bytes with 0s
+            let masterCopy2 := and(sload(0), 0xffffffffffffffffffffffffffffffffffffffff)
+            // 0xa619486e == keccak("masterCopy2()"). The value is right padded to 32-bytes with 0s
```

Comment:
this should stay `keccak("masterCopy()")`

Reply:
Corrected.